From 893b644fc8c840d0a29974c90e0a80f7ce8146ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 14 Feb 2016 10:21:02 +0100
Subject: [PATCH] tests/th: Update to work with perl-5.16+

The 'getopts.pl' module (dating back to perl4) was finally nuked in
perl-5.16+, therefore making it impossible to run tests in posh. Use
the perl5 Getopt::Std module instead.
---
 tests/th | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/th b/tests/th
index 72eb812..920d54c 100755
--- a/tests/th
+++ b/tests/th
@@ -130,7 +130,7 @@
 
 $os = defined $^O ? $^O : 'unknown';
 
-require 'getopts.pl';
+use Getopt::Std;
 
 ($prog = $0) =~ s#.*/##;
 
@@ -200,7 +200,7 @@ $nxpassed = 0;
 
 %known_tests = ();
 
-if (!&Getopts('C:p:Ps:t:ve:')) {
+if (!&getopts('C:p:Ps:t:ve:')) {
     print STDERR $Usage;
     exit 1;
 }
-- 
2.7.1

